php/php.editor/test/unit/src/org/netbeans/modules/php/editor/indent/PHPFormatterWrappingTest.java

```diff
@@ -686,6 +686,12 @@ public void testCoalescingOp03a() throws Exception {
         reformatFileContents("testfiles/formatting/wrapping/coalescingOp03a.php", options);
     }
 
+    public void testCoalescingOp03b() throws Exception {
```

Comment:
Maybe, I overwrote this method before I copy this method (`testCoalescingOp03b()`->`testCoalescingOp04a()`)

Reply:
Not sure what you mean exactly, sorry...

Reply:
When I added the next unit test(`testCoalescingOp04a()`), I've removed `testCoalescingOp03b()` accidentally.  It seems that I didn't notice that and I've committed and pushed it...

Reply:
I see now, thanks!